### PR TITLE
Tlmblgis 22 2

### PR DESCRIPTION
**Pull request by gsilver**

### CODE REVIEW

**Comment by lsloan** [comment]
I'm curious about the addition of the many comments.  Did you decide to add them because the JS code is difficult to understand?  Is it because of the nature of JS' anonymous functions, etc?  Would different, more specific symbol names help?

**Comment by gsilver** [comment]
It does seem overdone. I hope the functions are named clearly, but also have to clarify many framework "things" that come with the framework, such as:

$scope.$on("leafletDirectiveMap.contextmenu", function(event, args)

Without the comment it is not immediately apparent that this is a tap and hold handler.

File: mbofui/app/js/utils.js [hunk 1]
```diff
@@ -43,7 +46,7 @@ var leafletize = function(data){
     return leafletList;
 };
 
-
+// placeholder
```
Comment (dovek) [inline mbofui/app/js/utils.js:49]: What is this a place holder for?

File: mbofui/app/js/controllers.js [hunk 2]
```diff
@@ -154,16 +183,23 @@ ocellus.controller('mapController', ['$scope', '$rootScope','$filter', '$timeout
...
     });
   };
+
+  // watch on changes to marker (event) collection and text input so that only events with
+  // the searched for text appear
   $scope.$watch("markers", function() {
     $scope.$watch('markerFilter', function(text) {
       $scope.markersFiltered = $filter('filter')($scope.markers, {
-        event: text
+        message: text
```
Comment (dovek) [inline mbofui/app/js/controllers.js:192]: I think now that the refactoring has been completed we shouldn't see any mistakes between event and message.

File: mbofui/app/js/controllers.js [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+  $scope.markers = [];
+  $scope.markersAll = [];
+
+  // map configuration object by object:
+  // center: zoom level and autoDiscover(set to user location)
+  // TODO: need to deal with users/devices that refuse to share location
+  // events: initial event object
+  // controls: added (but not enabled: draw)
+  // layers: get baselayers (tiling options) from app.js, set overlays and draw options
+  // awesomeMarkerIcon: set a default. We will be setting different color/icons based on type of event later on
```
Comment (dovek) [inline mbofui/app/js/controllers.js:20]: Although these variable names seem relatively easy to understand, I definitely appreciate the comments.

File: mbofui/app/js/utils.js (hunk 1 above)
Comment (gsilver) [inline mbofui/app/js/utils.js:49]: An Angular template will be providing all the content of the popup - for now this will do.

**Pull request merged**